README.md

```diff
@@ -12,6 +12,7 @@
 	* [Force Unwrap](#force-unwrap)
 	* [Access Modifiers](#access-modifiers)
 	* [Type Declarations](#type-declarations)
+	* [Type Inference](#type-inference)
```

Comment:
Github won't let me comment on something not in this PR but Standards doesn't have its own link. It's bothering me